tests/tests_suite_test.go

```diff
@@ -168,8 +168,20 @@ func cancel(url, username, password string) {
 	// log in to the account
 	login(url, username, password)
 
+	// remove any existing test-* apps
```

Comment:
Only suggestion is if there are other places besides `cancel` that might want to remove the test apps in such fashion, then maybe we can extract this logic into its own helper func, `destroy` or `destroyAllApps`.